web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.utils.ts

```diff
@@ -35,25 +41,86 @@ export const getDisplayAdmin = (address?: string): User | undefined => {
   };
 };
 
-export const getDisplayDeveloper = (
-  metadata?: Partial<ProjectMetadataIntersectionLD>,
-  party?: Maybe<PartyFieldsFragment>,
-): User | undefined => {
-  if (!metadata) return;
-  if (metadata?.['regen:projectDeveloper']) {
-    return getDisplayParty('regen:projectDeveloper', metadata, party);
-  }
-  // Possibly temporary: CFC case
-  const projectOperator = (metadata as Partial<CFCProjectMetadataLD>)?.[
-    'regen:projectOperator'
-  ];
-  if (projectOperator) {
-    return {
-      name: projectOperator?.['schema:name'],
-      type: 'ORGANIZATION',
-      link: projectOperator?.['schema:url']?.['@value'] || '',
-    };
-  }
+type AnchoredMetadata = {
+  projectName?: string;
+  area?: string;
+  areaUnit?: string;
+  placeName?: string;
+};
+
+export const getAnchoredMetadata = (
+  anchoredMetadata?: Partial<AnchoredProjectMetadataIntersectionLD>,
+): AnchoredMetadata => {
+  const projectName = anchoredMetadata?.['schema:name'];
+  const projectSize = anchoredMetadata?.['regen:projectSize'];
+  const area = projectSize?.['qudt:numericValue']?.['@value'];
+  const unit = projectSize?.['qudt:unit']?.['@value'];
+  const areaUnit = getAreaUnit(unit as qudtUnit);
+  const placeName =
+    anchoredMetadata?.['schema:location']?.['place_name'] ??
+    anchoredMetadata?.['schema:location']?.['geojson:place_name'];
+
+  return { projectName, area, areaUnit, placeName };
+};
+
+type UnanchoredMetadata = {
+  videoURL?: string | null;
+  glanceText?: string[];
+  primaryDescription?: string;
+  quote?: ProjectQuote;
+  landStewardPhoto?: string | null;
+  landStewardStoryTitle?: string | null;
+  landStewardStory?: string | null;
+};
+
+export const getUnanchoredMetadata = (
```

Comment:
similar nit here; in both of these cases we already "got" the metadata, and now we are parsing it but no very strong opinion here.